01-ruby-fundamentals/keyword-arguments.md

```diff
@@ -157,39 +205,38 @@ coffee_price(:drip, :tall, cold: true)
 coffee_price(:cappuccino, :grande)
 ```
 
-The only things we've changed are the parameters in the method signature, the body of the method remains the same. The end result should resemble [this gist](https://gist.github.com/droberts-ada/07714ad0e2a3bc5a60c50d6a78e02f15).
-
-There are a few things to note here:
-- In the method signature, each keyword argument comes with a _default value_
-  - The syntax is similar to keys and values in a hash
-- When you invoke a method, keyword arguments are optional
-  - If you omit one, the value from the method signature will be used instead
-- When a method is invoked, any keyword arguments are passed by name (a.k.a. _keyword_)
-- Keyword arguments can be passed in any order.
-- Keyword arguments can co-exist with regular arguments (also called _positional arguments_)
-  - Positional arguments must come first, both in the definition and when the method is called
-  - All positional arguments must be supplied for every call
```

Comment:
Are we sure we want to get rid of this list?  It seems like the repetition might be good.

Reply:
Ah, it hadn't occurred to me that we'd want repetition within the same lesson/markdown but I'm happy to bring it back.
Much of it would need to be updated (lots of conflating between optional and keyword arguments) but I can certainly do that.

Reply:
Not all of these still apply, and some of them have or could be moved elsewhere. I would go through and make sure that none of the points are lost, and that they're all in the right spots.

Reply:
It is all covered elsewhere, a couple weren't covered until the summary so I'll add them after they're introduced as well.